web/packages/test/scripts/build-binary.sh

```diff
@@ -58,27 +58,26 @@ build_relayer() {
     cp $relay_bin "$output_bin_dir"
 }
 
-hack_beacon_client() {
+set_slot_time() {
+    local new_value=$1
     echo "Hack lodestar for faster slot time"
-    local preset_minimal_config_file="$root_dir/lodestar/packages/config/src/chainConfig/presets/minimal.ts"
+    local preset_mainnet_config_file="$root_dir/lodestar/packages/config/src/chainConfig/presets/mainnet.ts"
     if [[ "$(uname)" == "Darwin" && -z "${IN_NIX_SHELL:-}" ]]; then
-        gsed -i "s/SECONDS_PER_SLOT: 6/SECONDS_PER_SLOT: 1/g" $preset_minimal_config_file
+        gsed -i "s/SECONDS_PER_SLOT: .*/SECONDS_PER_SLOT: $new_value,/g" $preset_mainnet_config_file
     else
-        sed -i "s/SECONDS_PER_SLOT: 6/SECONDS_PER_SLOT: 1/g" $preset_minimal_config_file
+        sed -i "s/SECONDS_PER_SLOT: .*/SECONDS_PER_SLOT: $new_value,/g" $preset_mainnet_config_file
     fi
 }
 
 build_lodestar() {
-    if [ ! -d "$root_dir/lodestar/packages/cli/lib" ]; then
-        pushd $root_dir/lodestar
-        if [ "$eth_fast_mode" == "true" ]; then
-            hack_beacon_client
-        fi
-        yarn install && yarn run build
-        popd
+    pushd $root_dir/lodestar
```

Comment:
I think it's fine to build lodestar on every run because it won't rebuild if there are no changes.

Reply:
Unfortunately, it did rebuild even no changes and cost almost 13s on my laptop.

Reply:
Yeah, that's a problem. Maybe we can add a bool flag/env in the script to control whether to force rebuild or not. But it's very nit-picky so leave it up to you.